### PR TITLE
[BQ] add bq button for map and timeline

### PR DESCRIPTION
**Pull request by chejennifer**
screen-recording-_23_
screen-recording-_24_

### CODE REVIEW

File: server/templates/tools/map.html [hunk 1]
```diff
@@ -34,4 +34,8 @@
 {% block footer %}
   <script src={{url_for('static', filename='map.js', t=config['VERSION'])}}></script>
   <script src="https://maps.googleapis.com/maps/api/js?key={{maps_api_key}}&libraries=places" async defer></script>
+{% endblock %}
+
+{% block subfooter_extra %}
+<button id="bq-link" class="btn-subfooter btn btn-primary" style="display: none">Big Query</button>
```
Comment (beets) [inline server/templates/tools/map.html:40]: Nit: Should be BigQuery without a space.

Should we add an action word -- Open in BigQuery?

```suggestion
<button id="bq-link" class="btn-subfooter btn btn-primary" style="display: none">BigQuery</button>
```

File: server/templates/tools/timeline.html [hunk 2]
```diff
@@ -37,6 +37,7 @@
 {% endblock %}
 
 {% block subfooter_extra %}
+<button id="bq-link" class="btn-subfooter btn btn-primary" style="display: none">Big Query</button>
```
Comment (beets) [inline server/templates/tools/timeline.html:40]: ditto

**Review by beets — APPROVED** [review]
this is nice! we should also add a todo to add a webdriver test for this. something to check that the queries will continue to work.

File: server/templates/tools/map.html (hunk 1 above)
Comment (chejennifer) [reply server/templates/tools/map.html]: done

File: server/templates/tools/timeline.html (hunk 2 above)
Comment (chejennifer) [reply server/templates/tools/timeline.html]: done

**Pull request merged**